var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -69,6 +69,8 @@ class PyScipy(PythonPackage):
     patch('https://git.sagemath.org/sage.git/plain/build/pkgs/scipy/patches/extern_decls.patch?id=711fe05025795e44b84233e065d240859ccae5bd',
           sha256='5433f60831cb554101520a8f8871ac5a32c95f7a971ccd68b69049535b106780', when='@1.2:1.5.3')
 
+    patch('scipy-clang.patch', when='%clang')
```

Comment:
Does this patch apply to all versions of py-scipy?

Reply:
I don't know. I I've only used it with 1.6.1, but my guess is that it's much broader than this specific version. Do you want me to find out which versions it applies to or just pin it to that version?

Reply:
It appears to have been introduced in `1.5.0`
https://github.com/scipy/scipy/blob/4c0fd79391e3b2ec2738bf85bb5dab366dcd12e4/scipy/special/_faddeeva.cxx#L133
It's not there in previous versions
https://github.com/scipy/scipy/blob/adc4f4f7bab120ccfab9383aba272954a0a12fb0/scipy/special/_faddeeva.cxx#L126

I'll apply the patch to `1.5.0` and up unless you have something else in mind.

Reply:
I added this change and tested `spack patch py-scipy@xxx %clang` for all the versions in `1.4.1:1.6.1`.